docs/responses.md

```diff
@@ -180,6 +180,31 @@ class App:
         await response(receive, send)
 ```
 
+If you have a standard generator or iterator (instead of an async generator), you can wrap it with `starlette.concurrency.iterator_to_async` to convert it to an async generator.
+
+Then you can use it with a `StreamingResponse`.
+
+This is specially useful for synchronous <a href="https://docs.python.org/3/glossary.html#term-file-like-object" target="_blank">file-like</a> or streaming objects, like those provided by cloud storage providers.
```

Comment:
@tomchristie This line might be too verbose, let me know if you want me to remove it (or anything else).